mrp_production_add_middle_stuff/wizard/wizard_addition.py

```diff
@@ -0,0 +1,60 @@
+
```

Comment:
* Please use simple headers in all py files following OCA templates
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3